refinery/ui/source/js/data-set-import/controllers/file-upload-s3.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('refineryDataSetImport')
+    .controller('RefineryFileUploadS3Ctrl', RefineryFileUploadS3Ctrl);
+
+  RefineryFileUploadS3Ctrl.$inject = ['$scope', 's3UploadService'];
+
+  function RefineryFileUploadS3Ctrl ($scope, s3UploadService) {
```

Comment:
keep things out of $scope, use vm syntax. https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y032